pynumdiff/utils/_chartrand_tvregdiff.py

```diff
@@ -1,5 +1,3 @@
-# pylint: skip-file
```

Comment:
Files can be skipped via directive in the `.pylintrc` file, so I moved the instruction there.